4.8/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
   && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
```

Comment:
I think this is actually the proper place to update Yarn. Please see https://github.com/nodejs/docker-node/issues/386#issuecomment-296825839 for context.